system/lib/pthread/library_pthread.c

```diff
@@ -85,33 +85,29 @@ void emscripten_thread_sleep(double msecs) {
   double now = emscripten_get_now();
   double target = now + msecs;
 
-  __pthread_testcancel(); // pthreads spec: sleep is a cancellation point, so must test if this
-                          // thread is cancelled during the sleep.
-  emscripten_current_thread_process_queued_calls();
-
   // If we have less than this many msecs left to wait, busy spin that instead.
-  const double minimumTimeSliceToSleep = 0.1;
+  const double minTimeSliceToSleep = 0.1;
```

Comment:
Lets use snake case since we are in C and drop the `const`.   e.g. `double min_time_slice_to_sleep`.    Lets make  that into its own cleanup PR?

Reply:
I'll split that into its own PR. Although, this file contains many camelCase variables. Maybe we should do that only for the code we touch in musl?

Reply:
I think this camelCase exists here because it was ported from JS where we do use camelCase.    How about doing a naming-only PR.. either before or after this one is fine with.

Reply:
Naming-only PR sounds good to me.